### PR TITLE
docs(main): fix rootfs docs

### PR DESCRIPTION
**Pull request by cuisongliu**

### CODE REVIEW

File: docs/api/cloudrootfs.md [hunk 1]
```diff
@@ -5,49 +5,53 @@ cloud rootfs will package all the dependencies refers to the kubernetes cluster
...
 .
 ├── bin
-│   ├── conntrack
-│   ├── crictl
-│   ├── kubeadm
-│   ├── kubectl
-│   ├── kubelet
-│   └── seautil
+│   ├── conntrack
+│   ├── containerd-rootless-setuptool.sh
```
Comment (fanux) [inline docs/api/cloudrootfs.md:9]: why not put script in scripts dir?

Comment (cuisongliu) [reply docs/api/cloudrootfs.md:9]: this is nerdctl script , nerdctl   download this scripts at same time

File: docs/api/cloudrootfs.md [hunk 2]
```diff
@@ -5,49 +5,53 @@ cloud rootfs will package all the dependencies refers to the kubernetes cluster
...
+│   ├── docker-proxy
+│   ├── rootlesskit
+│   ├── rootlesskit-docker-proxy
+│   ├── runc
+│   └── vpnkit
 ├── etc
-│   ├── 10-kubeadm.conf
-│   ├── Clusterfile # image default Clusterfile
-│   ├── docker.service
-│   ├── kubeadm.yaml.tmpl
```
Comment (fanux) [inline docs/api/cloudrootfs.md:34]: we need kubeadm yaml later

File: docs/api/cloudrootfs.md [hunk 3]
```diff
@@ -5,49 +5,53 @@ cloud rootfs will package all the dependencies refers to the kubernetes cluster
...
-│   └── kubelet.service
+│   ├── 10-kubeadm.conf
+│   ├── Clusterfile  # image default Clusterfile
+│   ├── daemon.json
+│   ├── docker.service
+│   ├── kubeadm-config.yaml
```
Comment (cuisongliu) [inline docs/api/cloudrootfs.md:38]: @fanux  i change name is  kubeadm-config.yaml ,it's same to kubeadm.yaml.tmpl

**Review by fanux — APPROVED** [review]

**Pull request merged**